config.yaml

```diff
@@ -153,49 +153,60 @@ commands:
         required: true
 templates:
   app_list:
-    slack: "{{name}} - {{owner_email}}"
-    hipchat: "{{name}} - {{owner_email}}"
+    body: |-
+      | Name | Owner |
+      | ---- | ----- |
+      ~each var=$results~
+      | ~$item.name~ | ~$item.owner_email~ |
+      ~end~
   app_info:
-    slack: |
-      Dynos: {{dynos}}
-      Git URL: {{git_url}}
-      Owner: {{owner_email}}
-      Region: {{region}}
-      Repo Size: {{repo_size}}
-      Slug Size: {{slug_size}}
-      Stack: {{stack}}
-      Web URL: {{web_url}}
-    hipchat: |
-      Dynos: {{dynos}}
-      Git URL: {{git_url}}
-      Owner: {{owner_email}}
-      Region: {{region}}
-      Repo Size: {{repo_size}}
-      Slug Size: {{slug_size}}
-      Stack: {{stack}}
-      Web URL: {{web_url}}
+    body: |-
```

Comment:
Yeah I was thinking we'd just have `table` and `list` functions baked into greenbar for this kind of thing.